specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/billingSubscription.json

```diff
@@ -612,131 +681,545 @@
+          "type": "object",
+          "$ref": "#/definitions/RenewalTermDetails"
+        },
+        "skuDescription": {
+          "description": "The SKU description of the product for which the subscription is purchased. This field is only available  for Microsoft Customer Agreement billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
         "skuId": {
-          "description": "The sku ID of the Azure plan for the subscription.",
+          "description": "The SKU ID of the product for which the subscription is purchased. This field is only available  for Microsoft Customer Agreement billing accounts.",
           "type": "string"
         },
-        "skuDescription": {
-          "description": "The sku description of the Azure plan for the subscription.",
+        "status": {
+          "description": "The status of the subscription. This field is not available for Enterprise Agreement billing accounts.",
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "Active",
+            "Disabled",
+            "Deleted",
+            "Warned",
+            "Expiring",
+            "Expired",
+            "AutoRenew",
+            "Cancelled",
+            "Suspended"
+          ],
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "BillingSubscriptionStatus",
+            "modelAsString": true
+          }
+        },
+        "subscriptionId": {
+          "description": "The ID of the usage-based subscription. This field is only available for usage-based subscriptions of Microsoft Customer Agreement Type billing accounts.",
           "type": "string",
           "readOnly": true
         },
         "suspensionReasons": {
-          "description": "The suspension reason for a subscription. Applies only to subscriptions in Microsoft Online Services Program billing accounts.",
+          "description": "The suspension reason for a subscription. This field is not available for Enterprise Agreement billing accounts.",
           "type": "array",
           "readOnly": true,
           "items": {
             "type": "string"
           }
+        },
+        "termDuration": {
+          "description": "The duration for which you can use the subscription. Example P1Y and P1M",
+          "type": "string"
+        },
+        "termStartDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Start date of the term in UTC time.",
+          "readOnly": true
+        },
+        "termEndDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "End date of the term in UTC time.",
+          "readOnly": true
         }
       }
     },
-    "Amount": {
-      "description": "The amount.",
+    "BillingProfileSpendingLimitDetails": {
+      "description": "Billing profile spending limit details of the product.",
```

Comment:
The spending limit details of the billing profiles to which the subscription is billed.